include/swift/AST/DiagnosticsSema.def

```diff
@@ -2682,8 +2682,12 @@ ERROR(tf_graph_attr_no_generic_functions,none,
       "@TensorFlowGraph cannot be applied to generic functions", ())
 
 // @TFParameter attribute
+ERROR(tfparameter_attr_tensorflow_not_imported,none,
+      "@%0 is enabled only when the 'TensorFlow' module is imported", (StringRef))
```

Comment:
"the 'TensorFlow' module is imported" -> "'TensorFlow' is imported"

Reply:
Done in https://github.com/apple/swift/pull/18190/commits/e81fd2ff7ed06eb7802a6f8b4443aae141205a5c.